### ISSUE TITLE
[Loc] Wrong Japanese translation for check command

### ISSUE THREAD

**Issue by yamachu** [dotnet/sdk#18949]
https://github.com/dotnet/sdk/blob/74fbe46589025209ab476023fa1e252afec7697d/src/Cli/dotnet/commands/dotnet-sdk/check/xlf/LocalizableStrings.ja.xlf#L5-L14

These translations shows `dotnet clean` commands.

Since the translation status is `needs-review-translation`, I think it will be corrected in the future, but since it shows different commands, it seems better to return the status to `new` and leave it in English.

If you are going to do a translation, for example, the following content would be better.

```diff
< .NET Clean コマンド
> .NET Check コマンド
```

and

```diff
< 以前に生成されたビルド出力を消去するコマンド。
> SDKの更新を確認します。
```

**Comment by wli3** [comment]
@cristianosuzuki77 could you look into this?

**Comment by nagilson** [comment]
@marcpopMSFT  I didn't even know this command existed, but it still hasn't been translated properly.

**Comment by marcpopMSFT** [comment]
yes but he usually doesn't check GH. Why don't you use https://aka.ms/ceLocBug and directly file the localization bug? I added a link to the loc help page to the new hire guide as I took me some time to find it.
